### ISSUE TITLE
[BUG] Debug message set as error

### ISSUE THREAD

**Issue by camaya** [carbonblack/carbon-black-cloud-sdk-python#107]
**I am seeing this behaviour on:**
 - OS: mac OS 10.15.7
 - Carbon Black Cloud Products: Audit & Remediation
 - Python Version: 3.9.1

**Describe the bug**
When using the live response api, the status requests are always being logged because `logging.error` is being used instead of `logging.debug`.

**Steps to Reproduce**

1. Execute the following script

```python
import logging

from cbc_sdk.rest_api import CBCloudAPI

logging.basicConfig(level=logging.ERROR)


def main():
    device_id = '{CHANGE_ME}'
    lr_api = CBCloudAPI(profile='live_response')
    lr_session = lr_api.live_response.request_session(device_id)
    lr_session.create_process('sleep 5', wait_for_completion=True)
    lr_session.close()


if __name__  == '__main__':
    main()
```
2. See many status log messages

**Expected behavior**
I expected no log messages

**Screenshots**
<img width="1183" alt="Screenshot 2021-01-19 at 15 55 36" src="https://user-images.githubusercontent.com/1669579/105051304-eb914280-5a6e-11eb-9986-350c0ab7c22c.png">

**Additional context**
Here's the line generating the logs https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/develop/src/cbc_sdk/live_response_api.py#L1298

**Comment by avanbrunt-cb** [comment]
@camaya Please correct if I am wrong but you set the logging level here `logging.basicConfig(level=logging.ERROR)` which is why the errors are being outputted

**Comment by camaya** [comment]
@avanbrunt-cb You're right! My point is that the log that is being written should not be marked as error, but as debug or info since it's an status update, ie, this line: ﻿﻿﻿https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/develop/src/cbc_sdk/live_response_api.py#L1299 should be a `log.debug()` call. At least that is what it looks like, I'm not sure if there's a legitimate reason to have it as error.

**Comment by avanbrunt-cb** [comment]
We have changes coming up for the Live Response API so this will be addressed when we update the SDK with those changes

**Comment by avanbrunt-cb** [comment]
Fixed in release v1.2.1
